.storybook/stories/alert/alert.stories.ts

```diff
@@ -39,6 +39,7 @@ const defaultParameters: Parameters = {
   argTypes: {
     // inputs
     clrAlertClosed: { defaultValue: false },
+    clrAlertLightweight: { defaultValue: false },
     clrAlertIcon: { defaultValue: '' },
     clrAlertType: { defaultValue: 'info', control: { type: 'radio', options: ALERT_TYPES } },
```

Comment:
We could have comments describing that "loading" and "neutral" alert types are only applicable for lightweight alerts

Reply:
Added label comments for `loading` and `unknown` alert types in story book in commit 72fe63.

![image](https://github.com/vmware-clarity/ng-clarity/assets/6692908/424c93f9-aaa7-46cd-aa23-31aae6212540)